.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
+	"directory": "./bower_components"
```

Comment:
For some reason, bower install, for me, installs everything at `src/bower_components`. This helps cement it at the right place, and we can remove this file once we move away from bower.